pkg/controller/cassandracluster/testdata/cassandracluster-2DC-default.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: "db.orange.com/v1alpha1"
+kind: "CassandraCluster"
+metadata:
+  name: cassandra-demo
+  labels:
+    cluster: k8s.pic
+  namespace: ns
+spec:
+  nodesPerRacks: 1
+  cassandraImage: cassandra:latest
+  restartCountBeforePodDeletion: 3
+  imagePullSecret:
+    name: advisedev # To authenticate on docker registry
+  rollingPartition: 0
+  service:
+    annotations:
+      external-dns.alpha.kubernetes.io/hostname: my.custom.domain.com.
+  pod:
+    annotations:
+      exemple.com/test: my.custom.annotation
+    tolerations:
+      - key: my_custom_taint
+        operator: Exists
+        effect: NoSchedule
+  dataCapacity: "3Gi"
+  dataStorageClass: "local-storage"
+  hardAntiAffinity: false
+  deletePVC: true
+  storageConfigs:
+    - mountPath: "/var/lib/cassandra/log"
+      name: "gc-logs"
+      pvcSpec:
+        accessModes:
+          - ReadWriteOnce
+        storageClassName: standard-wait
+        resources:
+          requests:
+            storage: 10Gi
+    - mountPath: "/var/log/cassandra"
+      name: "cassandra-logs"
+      pvcSpec:
+        accessModes:
+          - ReadWriteOnce
+        storageClassName: standard-wait
+        resources:
+          requests:
+            storage: 10Gi
+  sidecarConfigs:
```

Comment:
@erdrix, we already have https://github.com/Orange-OpenSource/casskop/blob/4a9b6210d4a75086e033c03f32aef7b553715b74/pkg/controller/cassandracluster/testdata/cassandracluster-2DC.yaml that has sidecars. Why that new file ?

Reply:
Because I want a CassandraCluster without liveness and readiness configurated to check in unit tests that the default values are well set in the statefulset.